BUILD.md

```diff
@@ -344,13 +344,13 @@ that the minimum supported compiler versions are GCC 4.8.2 and Clang 3.4,
 although earlier versions may work. It should be straightforward to adapt this
 repository to other Linux distributions.
 
+The continuous integration tools use [CMake 3.12.4](https://github.com/Kitware/CMake/releases/tag/v3.12.4) for Linux
+
 #### Required Package List
 
     sudo apt-get install git cmake build-essential libx11-xcb-dev \
         libxkbcommon-dev libwayland-dev libxrandr-dev
 
-*Note: CI uses CMake 3.12.4*
```

Comment:
There is a note similar to this in the MacOS section on line 706 that also needs to be removed.

Reply:
Thanks, fixed.